README.md

```diff
@@ -2,28 +2,28 @@
 
 ![Amazon ECS logo](doc/ecs.png "Amazon ECS")
 
-![Build Status](https://github.com/aws/amazon-ecs-agent/workflows/Build/badge.svg?branch=dev)
-
 The Amazon ECS Container Agent is a component of Amazon Elastic Container Service
 ([Amazon ECS](http://aws.amazon.com/ecs/)) and is responsible for managing containers on behalf of Amazon ECS.
 
+This repository comes with ECS-Init, which is a [systemd](http://www.freedesktop.org/wiki/Software/systemd/) based service to support the Amazon ECS Container Agent and keep it running. It is used for systems that utilize `systemd` as init systems and is packaged as deb or rpm. The source for ECS-Init is available in this repository at `./ecs-init` while the packaging is available at `./packaging`.
```

Comment:
Non-blocking; should we add a comment to the `ecs-init` repo to tell visitors that it has been merged into the ECS agent repo too?

Reply:
We might deprecate repo and as part of that notify that. For now, we do have this [proposal](https://github.com/aws/amazon-ecs-init/issues/482) to notify customers.

Reply:
Ah, perfect.